distribution/src/resources/config-tool/templates/conf/carbon.xml.j2

```diff
@@ -89,8 +89,11 @@
     <!--
        Webapp context root of WSO2 Carbon management console.
     -->
+    {% if server.webcontexroot is defined %}
+    <WebContextRoot>{{ server.webcontexroot }}</WebContextRoot>
```

Comment:
Shall we make the parameter with an underscore. `web_context_root`.  Other places use the same convention.

Reply:
Sure, changed in https://github.com/wso2/micro-integrator/pull/2851/commits/2a8c568673cdbec1884ec256934bf6ed25cfb458